dev-tools/test.sh

```diff
@@ -18,19 +18,23 @@ export INTEGREAT_CMS_SUMM_AI_API_KEY="dummy"
 
 # Disable linkcheck listeners during testing
 export INTEGREAT_CMS_LINKCHECK_DISABLE_LISTENERS=1
+NUMPROCESSES="--numprocesses=auto"
 
 # Parse given command line arguments
 while [[ $# -gt 0 ]]; do
     case $1 in
         # If only tests affected by recent changed should be run, --changed can be passed as a flag
         --changed) CHANGED=1;shift;;
+        # Verbosity for pytest (one extra v to counteract --quiet)
+        # also disable multiple processes by xdist so diffs are full terminal width
+        -v|-vv|-vvv|-vvvv) VERBOSITY="$1v";NUMPROCESSES="";shift;;
         # If only particular tests should be run, test path can be passed as CLI argument
         *) TEST_PATH=$1;shift;;
     esac
 done
 
 # The default pytests args we use
-PYTEST_ARGS=("--quiet" "--disable-warnings" "--numprocesses=auto")
+PYTEST_ARGS=("--quiet" "$VERBOSITY" "--disable-warnings" "$NUMPROCESSES" "--color=yes")
```

Comment:
```suggestion
PYTEST_ARGS=("--disable-warnings" "--color=yes")

if [[ -n "${VERBOSITY}" ]]; then
    PYTEST_ARGS+=("$VERBOSITY")
else
    PYTEST_ARGS+=("--quiet" "--numprocesses=auto")
fi
```

I think it would make sense to only start with the arguments which are present for all test calls and then append all additional arguments based on the command line input

Reply:
Ah, I tried to group related arguments (quiet/verbose). Will do!